src/main/extension/extension_install.cpp

```diff
@@ -15,6 +15,9 @@
 
 #include <fstream>
 
+#define STRINGIFY(x) #x
```

Comment:
I don't think this should be necessary - if this is required this likely means the define is not being passed in correctly as a string literal. Fixing this and making sure the define is passed in correctly as a string literal should also solve the issue where `stripTrailingNulls` is required.